src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -33,5 +35,31 @@ object DeeperCommandExecutor : IdofrontCommandExecutor() {
                     sender.info("${player.name} is in section ${section.key}")
             }
         }
+        "syncedtime" {
+            "set"{
+                val time by intArg()
+                action {
+                    DeeperConfig.data.time.mainWorld?.let {
+                        it.time = time.toLong()
+                    } ?: return@action
+
+                    DeeperConfig.data.time.syncedWorlds.forEach {
+                        it.key.time = time.toLong() + it.value
```

Comment:
Just btw, you can do `forEach { (key, value) -> ... }` to name the key and value so they're a little more clear. Maybe use `(world, offset)` here.